modules/Makefile.circleci

```diff
@@ -16,13 +16,10 @@ circle\:deps:
 
 ## Tag using branch + build num and push to registry (CircleCI)
 circle\:tag:
-	@$(call assert_set,CIRCLE_BRANCH)
-	@$(call assert_set,CIRCLE_BUILD_NUM)
+	@$(call assert_set,BUILD)
```

Comment:
True, although it is also being substituted into the docker build `build` argument -- which I assume percolates into internal versioning for things like sentry / ops logging?